python/operator/bot/operator_bot.py

```diff
@@ -21,9 +21,8 @@ class CHAT:
     UserSession = 'Chat.V1:UserSession'
 
 
-class SLACK_INTEGRATION:
-    class OUTBOUND_MESSAGE:
-        OutboundMessage = 'SlackIntegration.OutboundMessage:OutboundMessage'
+class SlackIntegration:
+    OutboundMessage = 'SlackIntegration.OutboundMessage:OutboundMessage'
```

Comment:
This should somehow be brought in via an import from the integration itself. (Long term, obv.)